.github/workflows/main.yml

```diff
@@ -85,6 +85,7 @@ jobs:
           make acceptance-cluster-delete
           make acceptance-cluster-setup
           export KUBECONFIG=$PWD/tmp/acceptance-kubeconfig
+          make prepare_ci_environment_k3d
```

Comment:
Now this page needs updating: https://github.com/epinio/epinio/blob/main/docs/developer/howtos/acceptance_tests.md

I wonder if you should explicitly set the 3 needed environment variables here, just to make it obvious that this is the way to configure the tests. Right now `EPINIO_CONFIG`, `EPINIO_BINARY` and `KUBECONFIG` are all set in 3 different places. It's not obvious that these are the 3 things needed in order to point the tests to an Epinio installation.

Maybe if we document it properly in the linked page, it will be enough.

Reply:
Done. Updated the docs